### PR TITLE
Added KT19 and SMART Quicklooks incl instrument status for flight 20240818a

### PR DESCRIPTION
**Pull request by KevinWolf-90**

### CODE REVIEW

File: orcestra_book/reports/HALO-20240816a.md [hunk 1]
```diff
@@ -193,8 +193,13 @@ specMACS RGB snapshots from polb (lower right) with EarthCARE underpass marked i
...
 ```{card}
-:img-top: ../figures/HALO-20240816a/HALO-20240816a-SMART-Quicklook.png
+:img-top: ../figures/HALO-20240816a/HALO_20240816a_SMART.png
 SMART
 ```
+```{card}
+:img-top: ../figures/HALO-20240816a/HALO_20240816a_KT-19.png
+KT-19
+```
+-
```
Comment (bjorn-stevens) [inline orcestra_book/reports/HALO-20240816a.md:203]: could this dashed line be the problem?

**Comment by bjorn-stevens** [comment]
Hi Kevin, this didn't build for me.  Did you try to build it with the changes?  Maybe ping Lukas for help.  Normally you can check this in your branch with `jb build orcestra_book`.  The error says something about all children of the card carousel needing to be of type card, and the only suspect I could find was a line I marked  in [orcestra_book/reports/HALO-20240816a.md](https://github.com/orcestra-campaign/book/pull/235#diff-1b08b4efab716c0e45ddda5be3b5e90f6773a3db9c547ab591c8a047dc5e837f) ... where a stray dash shows up.  Maybe deleting this will allow things to build?

**Comment by KevinWolf-90** [comment]
Dear Bjorn. The file names were different than the file names in the md files. I made an inconsistent change at the end.
The plots are renamed and now it should work.
I have not tried to build it. 
I edited and uploaded the files via the github page. Note sure where to run the comment you sent me.

**Comment by bjorn-stevens** [comment]
hmmm ... now I see two errors. 

```bash 
/home/runner/work/book/book/orcestra_book/reports/HALO-20240811a.md:: WARNING: image file not readable: figures/HALO-20240811a/HALO-20240811a-SMART-Quicklook.png
/home/runner/work/book/book/orcestra_book/reports/HALO-20240816a.md:203: WARNING: All children of a 'card-carousel' should be 'card' [design.card]
```

 We should sit together or with @lkluft tomorrow  to try and sort this out.

**Comment by KevinWolf-90** [comment]
There was still an issue with Flight report 20240811. One of the pis already included the smart quicklook. The filename was incorrect in the md file. Now it should be correct.
I also copied the repo to a local folder and used jupiter-book but get all kinds of error messages that are not related to the uploaded quicklooks and reports.
In case the problem still persists - when can we meet at the hotel?

File: orcestra_book/reports/HALO-20240816a.md (hunk 1 above)
Comment (lkluft) [reply orcestra_book/reports/HALO-20240816a.md:203]: Yes, that dashed line is the problem.

```suggestion
```

**Comment by KevinWolf-90** [comment]
@lkluft. Can you try again, please.

I removed the dash and changed some file names.

Best,
Kevin.

**Pull request merged**